src/Compilers/CSharp/Portable/Symbols/Source/SourceUserDefinedOperatorSymbolBase.cs

```diff
@@ -649,10 +649,13 @@ internal sealed override void AfterAddingTypeMembersChecks(ConversionsBase conve
 
             ParameterHelpers.EnsureIsReadOnlyAttributeExists(Parameters, diagnostics, modifyCompilation: true);
 
+            var location = ReturnTypeSyntax.Location;
+            ReturnType.ReportAnnotatedUnconstrainedTypeParameterIfAny(location, diagnostics);
             if (ReturnType.ContainsNullableReferenceTypes())
             {
-                this.DeclaringCompilation.EnsureNullableAttributeExists(diagnostics, ReturnTypeSyntax.Location, modifyCompilation: true);
+                this.DeclaringCompilation.EnsureNullableAttributeExists(diagnostics, location, modifyCompilation: true);
             }
+            ParameterHelpers.ReportAnnotatedUnconstrainedTypeParameters(Parameters, diagnostics);
```

Comment:
>ReportAnnotatedUnconstrainedTypeParameters [](start = 29, length = 42)

Is the operator case covered by test?

Reply:
Maybe add a PROTOTYPE comment to cover

---
In reply to: [205892348](https://github.com/dotnet/roslyn/pull/28804#discussion_r205892348) [](ancestors = 205892348)

Reply:
The operator case is covered by `NullableT_Members`. I've added a constructor case to that test.

---
In reply to: [205909782](https://github.com/dotnet/roslyn/pull/28804#discussion_r205909782) [](ancestors = 205909782,205892348)